### PR TITLE
New Crowdin translations

### PR DESCRIPTION
**Pull request by timothyylim**
Enjoy!

### CODE REVIEW

File: language/de.json [hunk 1]
```diff
@@ -1,292 +1,321 @@
...
-                      "label": "Als Button anzeigen"
+                      
                     },
                     {
-                      "label": "Titel"
+                      
                     },
                     {
-                      "label": "Gehe zu Schaubild"
+                      "label":"Kommentare",
```
Comment (otacke) [inline language/de.json]: Will semantics.json also be changed? As of this position, quite some translations have changed and don't match with semantics. There, "Title" is followed by "Go To slide" which corresponds to the old translation ("Titel" and "Gehe zu Schaubild"). The same seems to be true for other translations.

Comment (timothyylim) [inline language/de.json]: @otacke , thanks for catching this. I'm going to look into it.

Comment (timothyylim) [inline language/de.json]: I think the new translations match with semantics. It appears that Crowdin has expanded the curly brackets but I don't think anything else is different (?)

Comment (timothyylim) [inline language/de.json]: @otacke , are you seeing any strange behaviour on your local site by any chance?

Comment (timothyylim) [inline language/de.json]: @otacke no worries then :) thank you for taking the time to check

Comment (otacke) [inline language/de.json]: @timothyylim Sorry, my business trip took longer than I expected. I'll check for strange behavior today.

Comment (otacke) [inline language/de.json]: It seems that in the editor, not everything is translated, e. g. most hover texts for the upper icon bar or the phrases connected to the keyword list ("Schlagwortliste" in German) although they seem to be contained in the language file. I have not had a look at the source code yet to figure out what might be the problem.

**Pull request closed without merging**